### DISCUSSION TITLE
[Bug Report] Address unreachable

### DISCUSSION THREAD

**Discussion by xWolpertinger01** [stashapp/stash#4941] [Q&A]
**Describe the bug**
When I try to connect and the stash-server is already running on another device I am having trouble to reach the address (ERR_ADDRESS_UNREACHABLE)

**To Reproduce**
1. Start Stash on another device
2. Trying to connect via local IP
3. Trying to connect
4. See error "ERR_ADDRESS_UNREACHABLE"

**Expected behavior**
I expected the connection to be reachable

**Screenshots**
If applicable, add screenshots to help explain your problem please ensure that your screenshots are SFW or at least appropriately censored.

**Stash Version: (from Settings -> About):**
v0.26.0
**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: Brave
 - Version: Brave Browser 125.1.66.118

**Additional context**
I did not change anything besides the proxy-, VPN-, and adblocker-settings to see if it's working afterwards. (Did not)

**Comment by feederbox826** [comment]
Sounds like you didn't allow it in your firewall settings

**Comment by WithoutPants** [comment]
This is usually some environmental issue. Can you connect to stash from the machine that it's running on?

Usual suspects are:
- stash server and connecting device are on different networks or subnets
- firewall rules on the server are preventing incoming connections on the applicable port
- stash is listening on a specific address/interface
